www/config.yml

```diff
@@ -1332,6 +1332,15 @@ chapters:
     latitude: -33.45
     longitude: -70.666667
   name: Santiago, Chile
+- email: panama@pyladies.com
+  image: Pyladies_Panama.png
+  location:
+    latitude: 8.54
+    longitude: 80.78
+  name: Panamaacute;
```

Comment:
@Kizuu Whatever you place here will display on the website card for the chapter. I would remove the trailing semicolon `:`

Reply:
Yes! I think the same about removing the trailing semicolon, the name that I wanted to show is "Panamá".

Reply:
Cool! Once you update the PR and it looks good, please @-mention me and I will be happy to merge. Thanks.

Reply:
@willingc Thank you so much!